code/game/gamemodes/changeling/powers/headcrab.dm

```diff
@@ -1,40 +1,42 @@
 /obj/effect/proc_holder/changeling/headcrab
-	name = "Last Resort"
```

Comment:
It looks like your editor changed all the tabs to spaces.

Reply:
Fixed I believe.